packages/jellyfish-api-core/__tests__/category/account.test.ts

```diff
@@ -479,4 +479,44 @@ describe('masternode', () => {
       })
     })
   })
+
+  describe('utxosToAccount', () => {
+    let address1: string
+    let address2: string
+
+    beforeAll(async () => {
+      address1 = await container.call('getnewaddress')
+      address2 = await container.call('getnewaddress')
+    })
+
+    it('should utxosToAccount', async () => {
+      const payload: any = {}
+      payload[address1] = '5@DFI'
+      payload[address2] = '5@DFI'
```

Comment:
you can state a note here regarding currently only support sending utxo to DFI account